src/useSearch.ts

```diff
@@ -59,7 +73,10 @@ export const useSearch = (
 
     const results = deserializeSearchResponse(responseJson.responseSet?.[0]) ?? [];
 
-    return compileDedupedResults(results);
+    return {
```

Comment:
Note that this is a breaking change so we'll bump the major with the next release.